agent/consul/stream/subscribe.proto

```diff
@@ -13,13 +15,17 @@ enum Topic {
 message SubscribeRequest {
     Topic topic = 1;
     string key = 2;
+    string token = 3;
```

Comment:
Do we need DC in here? I've not thought about cross-DC forwarding at all yet but we probably should. I suggest making a note of it on the project board in Asana and we can tackle that separately.